### PR TITLE
New buildium release

### PR DESCRIPTION
**Pull request by vianna274**

### CODE REVIEW

File: bin/swagger.json=
Comment (Andy9822) [inline bin/swagger.json=:1]: what is this `.json=` file? is this right @vianna274 ?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        ruby-version: ["3.1", "3.0", "2.7", "2.6"]
+        ruby-version: ["3.1", "3.0"]
```
Comment (Andy9822) [inline .github/workflows/ci.yml:15]: giving it some more thought now...
this gem is not only ours, we've actually published it to rubygems so it fells wrong to simply break its compatibility with previous Ruby version for no reason

Comment (Andy9822) [reply .github/workflows/ci.yml:15]: hey @ajGingrich , have you ever re-generated the Buildium SDK? asking to know if you have the operan-api-generator set up in your machine

Comment (Andy9822) [reply .github/workflows/ci.yml:15]: I'm afraid we are causing too many different and big changes when only wanted to re-generate the SDK but got blocked by the openapi-generator CLI failing on both Leo's and my machines :/

Comment (ajGingrich) [reply .github/workflows/ci.yml:15]: I've never generated any of the integrations SDKs 😢

Comment (noordean) [reply .github/workflows/ci.yml]: I think I have done that before [here](https://github.com/LeadSimple/buildium/pull/6), that was in my first few months here, I can't completely remember how I did it though but I didn't have this many changes. Would you want me to try something?

Comment (vianna274) [reply .github/workflows/ci.yml]: Fixed guys.

**Review by Andy9822 — APPROVED** [review]

**Pull request merged**